bot.py

```diff
@@ -106,7 +116,7 @@ async def updatethings(after,topic):
     if guid is not None:
         LOG.msg(f'found guid: {guid}, linking to channel: {after.id}')
         await after.send(f'Now relaying game log for {guid} to this channel. Good luck!')
-        r = requests.post(f'http://localhost:8000/api/game/{guid}/link/{after.id}')
+        r = requests.post(f'http://{DJANGO_HOST}:{DJANGO_PORT}/api/game/{guid}/link/{after.id}', headers=auth_header)        LOG.msg(r)
```

Comment:
```suggestion
        r = requests.post(f'http://{DJANGO_HOST}:{DJANGO_PORT}/api/game/{guid}/link/{after.id}', headers=auth_header)
```

duplicated instance of `LOG.msg(r)`